data/templates/esx-boot-cfg

```diff
@@ -0,0 +1,7 @@
+bootstate=0
+title=Loading ESXi installer
+kernel=<%=tbootFile%>
+kernelopt=runweasel formatwithmbr com1_baud=115200 com1_Port=<%=comportaddress%> tty2Port=<%=comport%> debugLogToSerial=1 logPort=<%=comport%> ks=http://<%=server%>:<%=port%>/api/common/templates/esx-ks
+modules=<%=moduleFiles%>
+build=
```

Comment:
The moduleFiles/tbootFile has already been added with repo before render (see the PR of on-core, os-repo-tool.js, function parseEsxBootCfgFile). Otherwise, I need double render and rewrite this template file under http data folder. Render function is currently not in on-core, on-tasks changes on-http file is not good architect design, and change on-http template needs restart. To solve all these problem need much work. so I choose the current solution, the simplest one.